src/libstd/sys/unix/mod.rs

```diff
@@ -164,6 +171,19 @@ pub fn cvt_r<T, F>(mut f: F) -> io::Result<T>
     }
 }
 
+#[cfg(target_os = "fuchsia")]
+pub fn mx_cvt<T>(t: T) -> io::Result<T> where T: TryInto<mx_status_t>+Copy {
```

Comment:
I think this is only used in the fuchsia process module, right? If so could this be moved into there for now? When we extract to `sys/fuchsia` we can put in `src/fuchsia/mod.rs` likely.